.github/workflows/validate_release.yml

```diff
@@ -0,0 +1,68 @@
+name: Validate release
+
+on:
+  workflow_call:
+    inputs:
+      path:
+        description: 'Go mod directory (e.g. "plugins/source/aws", "plugins/destination/csv", "cli")'
+        type:        string
+        required:    true
+      runner:
+        description: 'Runner to be used (default: "ubuntu-latest")'
+        type:        string
+        required:    false
+        default:     'ubuntu-latest'
+      CGO_ENABLED:
+        description: 'CGO_ENABLED value to be used (default: 0)'
+        type:        number
+        required:    false
+        default:     0
+
```

Comment:
```suggestion
```

I don't think this is used anywhere, can you please verify?

Reply:
yep, leftover code